Bugsnag/BugsnagSessionTracker.h

```diff
@@ -17,7 +17,7 @@ NS_ASSUME_NONNULL_BEGIN
 
 typedef void (^SessionTrackerCallback)(BugsnagSession *_Nullable newSession);
 
-extern NSString *const BSGSessionUpdateNotification;
+static NSNotificationName const BSGSessionUpdateNotification = @"BugsnagSessionChanged";
```

Comment:
I'm reluctant to do that because we don't have any other notification names to put in such a header.

Also I think declaring the notification in this header makes it more discoverable. If the notification were declared elsewhere, a developer looking at this header wouldn't realise there's a way to observe the session tracker.